modules/schedule/index.js

```diff
@@ -1,22 +1,24 @@
-var later  = require('later');
-var moment = require('moment');
-var sugar  = require('sugar');
-var hash = require('json-hash'); 
+/* eslint-disable no-param-reassign */
+const later = require('later');
+const Moment = require('moment');
+const hash = require('json-hash');
+require('sugar');
+require('moment-duration-format');
 
-require("moment-duration-format");
+let bot;
+let log;
 
-var bot;
+let minimumCreationDelay = 0;
+let minimumInterval = 60;
+let noCommands = false;
+let digestLength = 4;
```

Comment:
curious; why the digestLength change?

Reply:
Back of the envelope math. I figured It's more likely schedule will break down due to performance issues before the threshold for a 1% collision chance.

Really: I got sick of testing and having to type out 8 letters ids to remove a schedule.